dask/dataframe/rolling.py

```diff
@@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
             self.rolling_kwargs, method_name, args, kwargs)}
         if self.rolling_kwargs['axis'] in [0, 'rows']:
             # roll in the partition direction (will need to access neighbor)
+            window = self.rolling_kwargs['window']
+            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
```

Comment:
> Thanks again for your help and guidance

I'm very happy for any help I can get on dask.dataframe.  There is a lot of value here and the load is a lot easier when there are a few of us working on it.